src/reanimated2/core.js

```diff
@@ -277,15 +268,23 @@ export function createAnimatedPropAdapter(adapter, nativeProps) {
   return adapter;
 }
 
-const capturableConsole = console;
-runOnUI(() => {
-  'worklet';
-  const console = {
-    debug: runOnJS(capturableConsole.debug),
-    log: runOnJS(capturableConsole.log),
-    warn: runOnJS(capturableConsole.warn),
-    error: runOnJS(capturableConsole.error),
-    info: runOnJS(capturableConsole.info),
-  };
-  _setGlobalConsole(console);
-})();
+if (!NativeReanimated.useOnlyV1) {
```

Comment:
Unrelated to this PR, but is it possible to expose this in a public API? My use case is to do feature detection regarding if I can use the Reanimated 2 APIs and then use the new implementation if possible.

Reply:
@Szymon20000 What do you think?

Reply:
Sure